js/mobile-menu.js

```diff
@@ -1,29 +1,29 @@
 document.querySelector('.menu-bars').addEventListener('click', () => {
   document.getElementById('desktop-nav').style.cssText = `
-    display: flex;
-    justify-content: space-around;
-    z-index: 50;
-    background-color: inherit;
-    position: fixed;
-    height: 100vh;
-    width: 100vw;
-    top: 0;
-    left: 0;
-    `;
+      display: flex;
+      justify-content: space-around;
+      z-index: 50;
+      background-color: inherit;
+      position: fixed;
+      height: 100vh;
+      width: 100vw;
+      top: 0;
+      left: 0;
+      `;
```

Comment:
- [ ] To make the UI appear as  per design, I suggest you make some minor changes to the margins to march your [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). This should be an easy fix for you 😃 Looking at how great you've perfomed on this project 👏🏽